config/software/delivery-cli.rb

```diff
@@ -15,7 +15,7 @@
 #
 
 name "delivery-cli"
-default_version "master"
+default_version "UW-31/delivery-cli-1.8.0-compatibility"
```

Comment:
Should this change be in a separate WIP commit that will be removed before merge?